### DISCUSSION TITLE
Bitwise implementation of Tic Tac Toe

### DISCUSSION THREAD

**Discussion by vinamogit** [stellar/sorobanathon#116] [General]
While practicing and learning Soroban, I made this Tic Tac Toe implementation. 

I hope that this is a small implementation while still understandable. It also implements Events for the Dapp to be able to display the running games live.

Repository: [tic-tac-toe](https://github.com/candela-network/tic-tac-toe)

**Comment by vinamogit** [comment]
Better with a public repo ;-)

**Comment by achallagundla** [comment]
Heya,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
